indy_common/config.py

```diff
@@ -51,6 +51,7 @@
 configStateDbName = 'config_state'
 attrDbName = 'attr_db'
 idrCacheDbName = 'idr_cache_db'
+tsRevocationDbName = "timestamp_revoc_db"
```

Comment:
We need to store timestamps of all state updates (not only revocation) to reduce the risk of malicious nodes sending incorrect old results.
So, it's better to rename it to `state_ts_db`